<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>

<head>
  <title>BrainOmics</title>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
  <!--[if lte IE 8]><script src="/team/assets/js/ie/html5shiv.js"></script><![endif]-->
  <link rel="stylesheet" href="/team/assets/css/main.css" />
  <!-- favicon -->
  <link rel="shortcut icon" type="image/png" href="https://vincentfrouin.github.io/team/assets/images/banner.png">
  <!--[if lte IE 9]><link rel="stylesheet" href="/team/assets/css/ie9.css" /><![endif]-->
  <!--[if lte IE 8]><link rel="stylesheet" href="/team/assets/css/ie8.css" /><![endif]-->
</head>


<body>

    <!-- Wrapper -->
    <div id="wrapper">

        <!-- Main -->
        <div id="main">
            <div class="inner">

                <!-- Header -->
  <header id="header">
    <a href="https://vincentfrouin.github.io/team" class="logo"><strong>Brainomics</strong> Neuroimaging Genetics</a>
    <ul class="icons">
      
      
      
      
      
      
      
      
      
      
    </ul>
  </header>


                <!-- Content -->
                <section>
                	<header class="main">
                		<h1>Variant annotation and prioritization</h1>
                	</header>

                  

                <p><img src="https://vincentfrouin.github.io/team/research/genetics/priorization_bioinformatics.png" alt="image-title-here" class="image" /></p>

<p>It has been found that most of the SNPs identified in GWAS are located in non coding region of the DNA, and yet they have an incidence on the trait they are associated to. A current hypothesis to explain this fact is that they are located in functional regions which can regulate gene expression (e.g. promoter, enhancer, Transcription Factor Binding Sites, regulators of chromatin opening). 
Therefore, we chose to use a variant prioritization approach based on DNA functional region matching, following Lu et al. method and prioritization tool “GenoWAP” [1,2,3]. This method integrates 22 genomic and epigenomic annotations to predict whole-genome functional annotation [1]. Then, it computes a score for each variant locus assessing its functionality given the annotations at this locus [3]. This scoring will prioritize variants and reveal the ones located in functional sites, thus having more chance to be causal variants for the trait. 
A visualization of prioritized GWAS signal compared to GWAS signal can be seen in the figure above.</p>

<p>Secondly, we used a declination of this approach, following Lu et al. method and tool “GenoSkyline” [2] which integrate 111 tissue specific epigenomic annotations to predict tissue-specific functional annotation. This allows to compute tissue-specific enrichment for GWAS signal, thus indicating in which tissue GWAS variants hit the most functional regions. This can identify relevant tissues and can further improve prioritization through integrating functional annotations of relevant tissue types.</p>

<p>Lu, Q. et al. (2015). A statistical 
framework to predict functional non-coding regions in the human genome through integrated analysis of annotation data. Sci. Rep., 5, 10576.</p>

<p>Lu, Q.,  et al. (2016). Integrative tissue- 
specific functional annotations in the human genome provide novel insights on many complex traits and improve signal prioritization in genome wide association studies. 
PLoS Genet, 12(4).</p>

<p>Lu, Q. et al. (2015). GenoWAP: GWAS signal prioritization through integrated analysis of genomic functional annotation. Bioinformatics, 32(4), 
542-548.</p>

<p>Thorvaldsdóttir, H. et al. (2013). Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. ‎Brief. Bioinform, 14(2), 178-192.</p>



              </div>
          </div>

          <!-- Sidebar -->
  <div id="sidebar">
    <div class="inner">

      <!-- Search -->
        <section id="search" class="alt">
          <form method="post" action="#">
            <input type="text" name="query" id="query" placeholder="Search" />
          </form>
        </section>

      <!-- Menu -->
        <nav id="menu">
          <header class="major">
            <h2>Menu</h2>
          </header>
          <ul>
            <li><a href="https://vincentfrouin.github.io/team/">Homepage</a></li>
            <li>
              <span class="opener">Research</span>
              <ul>
                <li><a href="/team/research/genetics/">Imaging Genetics</a></li>
                <li><a href="/team/research/methods/">Methodology</a></li>
                <li><a href="/team/research/applications/">Applications</a></li>
              </ul>
            </li>
            <li>
              <span class="opener">Publications</span>
              <ul>
                <li><a href="/team/bibliography/selected.html">Selection</a></li>
                <li><a href="/team/bibliography/other.html">Present-2015</a></li>
              </ul>
            </li>
            <li><a href="/team/group_members.html">Members</a></li>
            <li><a href="/team/collaborations/index.html">Collaborations</a></li>
            <li><a href="/team/opportunities/index.html">Opportunities</a></li>
            <li><a href="/team/venue.html">Contact</a></li>
          </ul>
        </nav>

      <!-- Search -->
        <section id="search" class="alt">
          <header class="major">
            <h2>News & Events</h2>
          </header>
          <ul style="list-style-type:none;">
			  <li><a href="https://paris-saclay-cds.github.io/autism_challenge/">N Guigui has been ranked in the IMPAC challenge (Oct.'18)</a></li>
			  <li><a href="">S Karkar presented at BIBM Madrid (Dec.'18)  </a></li>
          </ul>
        </section>


      <!-- Footer -->
        <footer id="footer">
          <p class="copyright"><a href="/team/mentionslegales.html">Mentions Legales. </a></p>
        </footer>

    </div>
  </div>


      </div>

      <!-- Scripts -->
  <script src="https://vincentfrouin.github.io/team/assets/js/jquery.min.js"></script>
  <script src="https://vincentfrouin.github.io/team/assets/js/skel.min.js"></script>
  <script src="https://vincentfrouin.github.io/team/assets/js/util.js"></script>
  <!--[if lte IE 8]><script src="https://vincentfrouin.github.io/team/assets/js/ie/respond.min.js"></script><![endif]-->
  <script src="https://vincentfrouin.github.io/team/assets/js/main.js"></script>


</body>

</html>
